README.md

```diff
@@ -110,6 +110,56 @@ iex> Neuron.query("""
     )
 ```
 
+You can do subscriptions:
+
+```elixir
+defmodule AddUserSubscription do
```

Comment:
I'm probably missing something as to why we've chosen this API, but do you think we could make it so that the user can start it as a regular supervisor/process?

I'm not familiar with `AbsintheWebSocket` but as far as I can gather we can always `import AbsintheWebSocket.Supervisor` and build our own supervisor. wdyt?

Reply:
Hi  @uesteibar `AbsintheWebSocket` does all the heavy lifting of  WebSocket protocol implementation.  This is why I decided to use this library.
`AbsintheWebSocket.Supervisor` does all the magic of  WebSocket subscriptions. This is why it must be used.

Reply:
Regarding testing. As we only add a wrapper around `AbsintheWebSocket.Supervisor`, `AbsintheWebSocket` has all the tests.
As a proof that  this wrapper works, I  added  it in this `Absinthe` repo  as a integration test:
https://github.com/karlosmid/zoom/tree/master

README.md has all instructions about `Absinthe`  subscriptions.

Reply:
@karlosmid I understand `AbsintheWebSocket.Supervisor` does the "magic", however I think we can still have our own supervisor by using `import` and thus allow folks to use them as regular processes.

Did you consider an API like the one defined in https://github.com/uesteibar/neuron/issues/33?

Reply:
regarding testing, I feel strongly that if this library offers an API, we do need tests that ensure it behaves as expected.

Reply:
Hi @uesteibar.

Do you agree that this is what I need to do:

- [ ] refactor https://github.com/karlosmid/neuron/blob/master/lib/neuron_subscription.ex to be aligned with https://github.com/uesteibar/neuron/issues/33
- [ ] add unit test that will mock `AbsintheWebSocket.Supervisor` and `AbsintheWebSocket.SubscriptionServer.subscribe` and test `handle_update` method call

Reply:
sounds good 👍 I'd also be fine if we test it without mocking `AbsintheWebSocket.Supervisor`, but it might be harder 😄

Reply:
@karlosmid did you get around to handling the 2 PR task lists?